README.md

```diff
@@ -1,145 +1,308 @@
-
-* `-fuse`: requires access to `/dev/fuse` as well as the `fusermount` binary. An
-  optional `-fuse_tmp` flag can specify where to place temporary files. The
-  directory indicated by `-dir` is mounted.
-* `-instances="project1:region:instance1,project3:region:instance1"`: A comma-separated list
-  of instances to open inside `-dir`. Also supports exposing a tcp port and renaming the default Unix Domain Sockets; see examples below.
-  Same list can be provided via INSTANCES environment variable, in case when both are provided - proxy will use command line flag.
-* `-instances_metadata=metadata_key`: Usable on [GCE](https://cloud.google.com/compute/docs/quickstart) only. The given [GCE metadata](https://cloud.google.com/compute/docs/metadata) key will be
-  polled for a list of instances to open in `-dir`. The metadata key is relative from `computeMetadata/v1/`. The format for the value is the same as the 'instances' flag. A hanging-poll strategy is used, meaning that changes to
-  the metadata value will be reflected in the `-dir` even while the proxy is
-  running. When an instance is removed from the list the corresponding socket
-  will be removed from `-dir` as well (unless it was also specified in
-  `-instances`), but any existing connections to this instance will NOT be
-  terminated.
-* `-ip_address_types=PUBLIC,PRIVATE`: A comma-delimited list of preferred IP
-  types for connecting to an instance. For example, setting this to PRIVATE will
-  force the proxy to connect to instances using an instance's associated private
-  IP. Defaults to `PUBLIC,PRIVATE`
-* `-term_timeout=30s`: How long to wait for connections to close before shutting
-  down the proxy. Defaults to 0.
-* `-skip_failed_instance_config`: Setting this flag will allow you to prevent the proxy from terminating when
-	some instance configurations could not be parsed and/or are unavailable.
-* `-log_debug_stdout=true`: This is to log non-error output to stdOut instead of stdErr. For example, if you don't want connection related messages to log as errors, set this flag to true. Defaults to false.
-* `-enable_iam_login`: This enables the proxy to use Cloud SQL IAM database authentication. This will cause the proxy to use IAM account credentials for database user authentication. 
+![CI][ci-badge]
+[![Go Reference][pkg-badge]][pkg-docs]
+
+The [Cloud SQL Proxy][proxy-page] is a binary that provides IAM-based
+authorization and encryption when connecting to a Cloud SQL instance.
+
+See the [Connecting Overview][connection-overview] page for more information on
+connecting to a Cloud SQL instance, or the [About the proxy][about-proxy] page
+for details on how the Cloud SQL proxy works.
+
+Note: The Proxy *cannot* provide a network path to a Cloud SQL instance if one
+is not already present (e.g., the proxy cannot access a VPC if it does not
+already have access).
+
+## Installation
+
+The proxy may be installed from the [releases page][releases].
+
```

Comment:
I think we should consider just picking what we think the most popular distribution (64x Linux?), and give users a command to download and set as an executable. I would then change this to something "Releases for additional OS's and architectures and be found on the releases pages).

Reply:
Done.